misc/launchd/org.nixos.nix-daemon.plist.in

```diff
@@ -4,6 +4,8 @@
   <dict>
     <key>EnvironmentVariables</key>
     <dict>
+      <key>NIX_SSL_CERT_FILE</key>
+      <string>/nix/var/nix/profiles/default/etc/ssl/certs/ca-bundle.crt</string>
```

Comment:
When NIX_SSL_CERT_FILE is unset, Nix should default to this:

https://github.com/NixOS/nix/blob/5080d4e7b2525d1656282c65a217a22ff8381df3/src/libstore/globals.cc#L47

see also https://github.com/NixOS/nix/commit/847f19a5f7a558252bbde9b4c70efa5f7fac1f4f

Reply:
Interesting. Something very weird must be going on then. Perhaps it was restarting the daemon rather than this setting that fixed it for me. I removed the environment variable and the daemon is still working ok.

Is it possible that the daemon is unable to access the certificate the first time it is started for some reason? Since it's clear from the other reports that I'm not the only one who had this problem with a fresh installation of Nix.

Reply:
That's a good point. `caFile` is set only once, when the daemon starts, so if the CA bundle is installed afterwards, the daemon needs to be restarted.